### ISSUE TITLE
Live preview issue

### ISSUE THREAD

**Issue by Mujaddadi** [adobe/brackets#12160]
I have a weird issue. While using live preview sometimes after editing the document when I save the file, the working page start showing twice on the browser windows. Means it becomes showing twice on the same window. If I refresh the browser it goes back to single. First I thought it's because of beautify extension conflict with some feature but now it is even showing after simple saving the document. I am using the latest chrome browser. Any Idea why is it happening?

**Comment by petetnt** [comment]
Similar to this old issue: https://github.com/adobe/brackets/issues/4985

There were some similar kinds of reports some time back but I couldn't find them with a quick googling.

**Comment by Mujaddadi** [comment]
so it was two years ago and it got fixed. It should not be happening again then. I encounter this issue randomly. I have been working in bracket since morning and so far it didn't show up. I will keep an eye on the changes I will made and message again if it comes back.

**Comment by petetnt** [comment]
Yeah I meant that the issue is similar but the cause is different.

**Comment by arashsa** [comment]
I experience this issue from time to time too. Specially with projects where I work with sass and there's a lot of precompiling going on. I don't know if that has something to do with it?

**Comment by KwrKy** [comment]
Just bumping this thread because I experienced the same issue for the first time today. 
Using an iMac, latest macOS, Brackets and Chrome.

**Comment by swmitra** [comment]
This issue still exist but in a particular workflow.  If you paste a chunk of markup in the source or do a select all and replace the whole markup this happens. Haven't exactly pin pointed the issue yet but looks like a bulk mutation is causing the instrumented code to append the nodes again. I will have a look at it sometime this week.
